action/action_deserializer.go

```diff
@@ -25,7 +25,14 @@ func (ad *Deserializer) SetEvmNetworkID(id uint32) *Deserializer {
 
 // ActionToSealedEnvelope converts protobuf to SealedEnvelope
 func (ad *Deserializer) ActionToSealedEnvelope(pbAct *iotextypes.Action) (*SealedEnvelope, error) {
-	var selp SealedEnvelope
-	err := selp.loadProto(pbAct, ad.evmNetworkID)
+	var (
+		selp SealedEnvelope
+		err  error
+	)
+	if pbAct.GetEncoding() == iotextypes.Encoding_ETHEREUM_CONTAINER {
+		err = selp.loadEvmTransaction(pbAct, ad.evmNetworkID)
+	} else {
```

Comment:
unified into `selp.loadProto`

Reply:
that kind of confused things a little bit conceptually
`loadEvmTransaction` is to temporarily house the evm tx bytes, and make the `Selp.Action()` expose correction funcs (like `Hash(), Cost(), IntrinsicGas()`), so actpool can work correctly 
the actual conversion (which calls `selp.loadProto`) will be called in `Unfold` inside workingset